src/components/List.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import uuid from 'react-uuid';
+
+import Task from './Task';
+
+function List({ list, onClick }) {
+  return (
+    <div>
+      { list.length < 1 ? (
```

Comment:
```suggestion
      { list.length === 0 ? (
```
위와 같이 처리하면 어떤가요? 좀 더 단순하게 이해 가능해질 것 같네요.

그리고 list가 넘어오지 않는 경우는 없을까요? 이 경우엔 `undefined`라서 에러가 발생할 것 같아서요 🤔

Reply:
optional chaining을 활용하여 undefined 케이스까지 validation  할 수 있게 처리하였습니다.

#### key point
- 다른 생각이 들지 않게 컨디션은 최대 단순하게 작성 요망
- length 뿐 아니라 null 또 undefined 일 경우의 exception을 대비

Reply:
처리했다 생각했지만 "You may need an additional loader to handle the result of these loaders" 라는 에러가 컴파일 나중에 나와 일단 보류 했습니다. 😭😭😭😭😭😭😭😭😭😭